pkg/controller/controller.go

```diff
@@ -326,6 +356,48 @@ func NewCSIProvisioner(client kubernetes.Interface,
 		extraCreateMetadata:                   extraCreateMetadata,
 		eventRecorder:                         eventRecorder,
 	}
+	if nodeDeployment != nil {
+		// The base delay configures the rate limiting which
+		// prevents concurrent external-provisioner instances
+		// to start working on a new PVC all at the
+		// sametime. This delay also influences the jitter, so the
+		// initial delay will be somewhere between 0 and
+		// baseDelay.
+		//
+		// It must be large enough that one instance has the chance to
+		// become owner of a PVC and get that change propagated to the
+		// other instances before those wake up to try that
+		// themselves.
+		//
+		// With a value of 10 seconds, when creating 5000
+		// volumes on a cluster with 50 instances only ~300
```

Comment:
Just to be sure, nr. of conflicts does not grow linearly with nr. of nodes, right? 5000 nodes != 30 000 conflicts, but much more. How much?

Reply:
I'll try to test that.